packages/experimental/kai/src/containers/FrameContainer/FrameContainer.tsx

```diff
@@ -16,5 +16,9 @@ export const FrameContainer: FC<{ frame: FrameDef }> = ({ frame }) => {
   }
 
   // TODO(burdon): Standardize container (bg, padding, centered, size, etc.)
-  return <Component />;
+  return (
+    <Suspense>
```

Comment:
It’s usually good practice to provide a fallback, e.g. `<Suspense fallback={<Loading/>}>…`

Reply:
Unless there's a really long block (>1s) then this results in flickering and as a user that means to me a broken app.
What I propose is a fallback that is silent for the first 200ms but then if triggered is visible for at lest 500ms. OR is much more subtle -- e.g., a 2px linear line at the bottom of the screen that is overlaid and familiar.
Anyway to discuss.